app/util/conf.py

```diff
@@ -50,5 +50,26 @@ def server_url(self):
         return f'{self.protocol}://{self.hostname}:{self.port}{self.postfix}'
 
 
+class BitbucketSettings:
```

Comment:
Can we try to get rid of settings copy/paste? Or create a ticket if this is not a trivial thing for some reason?